treeherder/model/derived/jobs.py

```diff
@@ -261,7 +261,8 @@ def cancel_all_jobs_for_push(self, requester, push_id):
 
         # Notify the build systems which created these jobs...
         for job in Job.objects.filter(push_id=push_id).exclude(state='completed'):
-            self._job_action_event(job.id, 'cancel', requester)
+            self._job_action_event(job.project_specific_id, 'cancel',
+                                   requester)
```

Comment:
This can probably stay on one line? (We don't have an 80 character cutoff - anything under 100 or so, or whatever works best for readability is fine :-))